Source/Libraries/openXDA.Nodes/Types/Analysis/AnalysisNode.cs

```diff
@@ -234,6 +234,7 @@ private void Process(FileGroup fileGroup, Meter meter)
             configurator(reader);
 
             MeterDataSet meterDataSet = reader.Parse(fileGroup);
+            FileGroupProcessingStatus processingStatus = FileGroupProcessingStatus.Failed;
```

Comment:
Previously if `(meterDataSet is null)`  it would got into `FileGroupProcessingStatus.Processed` state while now it will remain in `Failed` that seems to be an issue. Technically I am not sure who frequent that is since it should not reach the `AnalysisNode` in most cases but I think there is a potential for issues here.